documentation/assemblies/metrics/assembly-metrics.adoc

```diff
@@ -11,6 +11,9 @@ You can use Prometheus and Grafana to monitor your Strimzi deployment.
 You can monitor your Strimzi deployment by viewing key metrics on dashboards and setting up alerts that trigger under certain conditions.
 Metrics are available for each of the components of Strimzi.
 
+If you are using `oauth` authentication or `opa` / `keycloak` authorization the collection of their specific metrics has to be explicitly enabled. For example, Kafka CR 'spec.kafka.listeners.authentication' and 'spec.kafka.authorization' have 'enableMetrics' property which needs to be set to 'true'.
```

Comment:
```suggestion
You can also collect metrics specific to `oauth` authentication and `opa` or `keycloak` authorization. 
You do this by setting the `enableMetrics' property to `true` in the listener configuration of the `Kafka` resource. 
For example, set `enableMetrics'  to `true` in `spec.kafka.listeners.authentication` and `spec.kafka.authorization`.
```

Reply:
Maybe to be consistent:

You do this by setting the `enableMetrics' property to `true` in the listener configuration or authorization configuration of the `Kafka` resource.